### PR TITLE
test: clarify skipped tests

### PR DESCRIPTION
**Pull request by danwinship**
got annoyed while trying to figure out what Tim changed in #1225 

Feel free to suggest improvements to the echo/sed/tr magic

Also, any info on the "???"s?

cc @dave-tucker @as-com

### CODE REVIEW

File: test/scripts/e2e-kind.sh [hunk 1]
```diff
@@ -10,8 +10,43 @@ export KUBECONFIG=${HOME}/admin.conf
...
+
+# REMOVE when k8s is updated to 1.18
+should allow ingress access from updated pod.+\[Feature:NetworkPolicy\]
+
+# NOT IMPLEMENTED; SEE DISCUSSION IN https://github.com/ovn-org/ovn-kubernetes/pull/1225
+named port.+\[Feature:NetworkPolicy\]
+
+# ???
+\[Feature:NoSNAT\]
+Services.+(ESIPP|cleanup finalizer|session affinity)
```
Comment (trozet) [inline test/scripts/e2e-kind.sh:42]: we don't support session affinity for sure. Not sure what ESIPP is or cleanup finalizer, will need to look

File: test/scripts/e2e-kind.sh [hunk 2]
```diff
@@ -10,8 +10,43 @@ export KUBECONFIG=${HOME}/admin.conf
...
+configMap nameserver
+ClusterDns \[Feature:Example\]
+should set TCP CLOSE_WAIT timeout
```
Comment (trozet) [inline test/scripts/e2e-kind.sh:45]: # kube-proxy test that relies on OS distribution version of conntrack

File: test/scripts/e2e-kind.sh [hunk 3]
```diff
@@ -10,8 +10,43 @@ export KUBECONFIG=${HOME}/admin.conf
...
+# REMOVE when k8s is updated to 1.18
+should allow ingress access from updated pod.+\[Feature:NetworkPolicy\]
+
+# NOT IMPLEMENTED; SEE DISCUSSION IN https://github.com/ovn-org/ovn-kubernetes/pull/1225
+named port.+\[Feature:NetworkPolicy\]
+
+# ???
+\[Feature:NoSNAT\]
+Services.+(ESIPP|cleanup finalizer|session affinity)
+configMap nameserver
```
Comment (trozet) [inline test/scripts/e2e-kind.sh:43]: no idea on this one or line 44
@as-com do you know why these tests were skipped?

File: test/scripts/e2e-kind.sh (hunk 1 above)
Comment (danwinship) [reply test/scripts/e2e-kind.sh:42]: oops, I had split session affinity off into its own line but forgot to remove it from this one

File: test/scripts/e2e-kind.sh (hunk 3 above)
Comment (as-com) [reply test/scripts/e2e-kind.sh:44]: [sig-network] NoSNAT [Feature:NoSNAT] [Slow] [It] Should be able to send traffic between Pods without SNAT 

[sig-network] Services [It] should have session affinity work for LoadBalancer service with ESIPP off [Slow] [DisabledForLargeClusters] 
[sig-network] Services [It] should be able to switch session affinity for LoadBalancer service with ESIPP on [Slow] [DisabledForLargeClusters] 
[sig-network] Services [It] should have session affinity work for LoadBalancer service with ESIPP on [Slow] [DisabledForLargeClusters] 
[sig-network] Services [It] should be able to switch session affinity for LoadBalancer service with ESIPP off [Slow] [DisabledForLargeClusters] 

**[sig-network] DNS configMap nameserver [Feature:Networking-IPv6] [LinuxOnly]** 
Change stubDomain [It] should be able to change stubDomain configuration [Slow][Serial] 
Forward PTR lookup [It] should forward PTR records lookup to upstream nameserver [Slow][Serial] 
Forward external name lookup [It] should forward externalname lookup to upstream nameserver [Slow][Serial] 
**[sig-network] DNS configMap nameserver [IPv4]** 
Forward external name lookup [It] should forward externalname lookup to upstream nameserver [Slow][Serial] 
Forward PTR lookup [It] should forward PTR records lookup to upstream nameserver [Slow][Serial] 

these are from [my notes](https://docs.google.com/document/d/1BOKGgsp7rwlP-Cn56DPcgbpkd1HADb6zlUA0ZrBa0OQ/edit#) on which tests are failing - does this help?

Comment (trozet) [reply test/scripts/e2e-kind.sh:44]: thanks @as-com it does help identify the individual test cases. For the DNS ones I have never looked at those, so not sure if those should pass or fail. Need to check.

**Comment by danwinship** [comment]
ok, filed #1316 to remind us to figure out what's up with the other skips

**Pull request merged**